### PR TITLE
Clean BPMN-Editor and add capability to enable/disable plugins at startup 

### PR DESCRIPTION
**Pull request by brucenguyen1**
This PR implements AP-2967 with the following changes:
- Rename 'ORYX' namespace to Apromore, all 'Oryx' word changed to 'Apromore' including stylesheets.
- Remove unsed files
- Remove those clearly unused configuration fields in Apromore.CONFIG and Apromore.I18N (there could be other unused fields to be checked and cleaned more carefully)
- Rename Apromore.Editor class to Apromore.EditorApp, and Apromore.Canvas class to Apromore.Editor: this naming is more appropriate.
- Remove unused fields in the param object passing to Apromore.EditorApp at startup, add enabledPlugins array to the param object to identify names of plugin to be enabled.
- Changes to Apromore.Plugins.Toolbar class to enable/disable buttons based on enabledPlugins.

This PR must be used with the PR https://github.com/apromore/ApromoreEE/pull/219.

Testing: 
- Open Apromore-Portal\src\main\webapp\macros\openModelInBPMNio.zul
- Add enabledPlugins param to the param object, e.g.
     new Apromore.EditorApp({
            xml: bpmnXML,
            callBack: callBack,
            id: editorContainerId,
            fullscreen: true,
            enabledPlugins: [window.Apromore.I18N.View.zoomIn, window.Apromore.I18N.View.zoomOut]})
- The name in enabledPlugins is the plugin name specified in the plugin constructor (those passing to facade.offer), go to each plugin in Apromore-BPMNEditor\src\main\scripts\plugins. 'window' must be used to indicate this global window namespace instead of Apromore module namespace.
- Compile, run and open BPMN-Editor, validate only the above buttons are enabled. If enabledPlugins is missing in the param object, all the buttons are enabled. If enabledPlugins is [], all the buttons are disabled.

### CODE REVIEW

**Review by dreamind — APPROVED** [review]

File: Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/bpmneditor/editor/css/theme_norm.css [hunk 1]
```diff
@@ -77,7 +77,7 @@ table-layout:fixed;
 	width:10px;
 }
 
-#oryx_editor_header {
+#Apromore_editor_header {
        background-image: url('../images/header_bg.small.gif');
```
Comment (dreamind) [inline Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/bpmneditor/editor/css/theme_norm.css:81]: It's not critical, but css selectors commonly use kebab-case. Here we use a mix of CamelCase and snake_case.

**Review by raboczi — APPROVED** [review]
Tried the suggested test in both Virgo and Karaf and it works nicely; approved.

File: Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/META-INF/spring/context.xml [hunk 2]
```diff
@@ -76,12 +64,4 @@
         </service-properties>
     </service>
 
-    <!-- Display the temporary files the PDF export servlet produces -->
-    <beans:bean id="tmpFileServlet" class="org.apromore.editor.server.TemporaryFileServlet"/>
-    <service ref="tmpFileServlet" interface="javax.servlet.http.HttpServlet">
-        <service-properties>
-            <beans:entry key="org.apromore.portal.servlet.pattern" value="/tmp/*" />
-        </service-properties>
-    </service>
```
Comment (raboczi) [inline Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/META-INF/spring/context.xml]: This servlet is required for the PDF export toolbar button to work.

**Review by raboczi — CHANGES_REQUESTED** [review]

File: Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/META-INF/spring/context.xml (hunk 2 above)
Comment (raboczi) [reply Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/META-INF/spring/context.xml]: Re-added this servlet and checked that PDF export works again.

**Review by raboczi — APPROVED** [review]
I replaced the servlet that displayed exported PDFs.

**Pull request merged**